be1-go/message/messagedata/consensus_failure.go

```diff
@@ -0,0 +1,53 @@
+package messagedata
+
+import (
+	"encoding/base64"
+
+	"golang.org/x/xerrors"
+)
+
+// ConsensusLearn defines a message data
+type ConsensusFailure struct {
+	Object     string `json:"object"`
+	Action     string `json:"action"`
+	InstanceID string `json:"instance_id"`
+	MessageID  string `json:"message_id"`
+
+	// CreatedAt is a Unix timestamp
+	CreatedAt int64 `json:"created_at"`
+}
+
+// Verify verifies that the ConsensusLearn message is correct
+func (message ConsensusFailure) Verify() error {
+	// verify that the instance id is base64URL encoded
+	if _, err := base64.URLEncoding.DecodeString(message.InstanceID); err != nil {
```

Comment:
We never use assignement in if statement for code clarity. Please do it in two phases.
Same bellow.